lib/Builtins.cpp

```diff
@@ -778,7 +778,8 @@ glsl::ExtInst Builtins::getDirectOrIndirectExtInstEnum(
 bool Builtins::BuiltinWithGenericPointer(StringRef name) {
   if (name.contains("fract") || name.contains("frexp") ||
       name.contains("modf") || name.contains("remquo") ||
-      name.contains("lgamma_r"))
+      name.contains("lgamma_r") || name.contains("vstore_half") ||
```

Comment:
Is this meant to only list function from libclc? Because there are other builtins that use the generic address space (e.g. atomics).

Reply:
Yes this is just for libclc, the goal being to keep function from libclc even though they might not be used.